homework-submission/week1-project/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Description from './Components/todoComponents.js'
+
+
+class App extends Component {
+  
+  constructor(props){
+    super(props);
+    this.state = {list: props.JSONList};
+  }
+
+  changeState = (id) => {
+    const JSONList = this.state.list;
+    JSONList[id].done = !JSONList[id].done;
+    this.setState({list: JSONList});
```

Comment:
That's not deep copy, you are directly changing the state
to do deep copy use spread operator or slice() method

` const JSONList = this.state.list.slice()` or `const JSONList ={ ...this.state.list}`